mne/beamformer/_dics.py

```diff
@@ -165,7 +170,10 @@ def make_dics(info, forward, csd, reg=0.05, label=None, pick_ori=None,
            in Large-Scale Cortical Networks Predicts Perception.
            Neuron (2011) vol 69 pp. 387-396.
            https://doi.org/10.1016/j.neuron.2010.12.027
-    .. [4] van Vliet, et al. (2018) Analysis of functional connectivity and
+    .. [4] Van Veen et al. Localization of brain electrical activity via
+           linearly constrained minimum variance spatial filtering.
+           Biomedical Engineering (1997) vol. 44 (9) pp. 867--880
```

Comment:
Ahh nice. Does this mean you have checked that these changes don't break your results?

Reply:
Yes. I've explicitly checked that the output of both `make_dics` and `make_lcmv` is identical to current master, as this is supposed to be just a refactoring. However, some bugs where exposed with some parameter combinations (see description of the PR at the very top), so the output in these cases is different.

Reply:
Excellent